sparse/compressed/convert.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np 
+from functools import reduce
+from operator import mul
+import numba
+
+def convert_prep(inds,shape,axisptr):

```

Comment:
Anything with heavy loops like this should have the numba decorator added. Python is slow, Numba makes it fast, at the expense of some stuff.